packages/tendermint-rpc/src/tendermint34/adaptor/responses.ts

```diff
@@ -534,13 +523,15 @@ interface RpcValidatorInfo {
   readonly address: string;
   readonly pub_key: RpcPubkey;
   readonly voting_power: string;
+  readonly proposer_priority?: string;
```

Comment:
Nice. Could you pull out the validator info changes into a separate PR? They seem to be staight forward and independent.